app/qml/components/MprisUi.qml

```diff
@@ -91,7 +91,7 @@ Column {
 
                     Label {
                         id: songLabel
-                        text: song ? song : ("<" + i18n("Nothing playing") + ">")
+                        text: ( title && artist ) ? (artist + " - " + title) : ("<" + i18n("Nothing playing") + ">")
```

Comment:
`artist` can be empty while `title` is not. So please add a branch for this.

Reply:
Done